### DISCUSSION TITLE
Render not allowing the creation of the service

### DISCUSSION THREAD

**Discussion by ToniG22** [Dadoum/Provision#99] [Discussions]
I don't know if it's something related to the amount of people creating this type of service, but as of now it does not allow me to deploy the server. Is there a way I can just make it run on a RaspBerry Pi hooked to my router?

![image](https://github.com/Dadoum/Provision/assets/61626139/e7dd234d-a0f7-4ffd-af08-0f90e2f77537)

**Comment by pieterdevriesch** [comment]
Same here. I guess I'll just run it at home.

**Comment by Dadoum** [comment]
I think you have to rename the repo to remove all the keywords that could link your fork to my project.

**Comment by pieterdevriesch** [comment]
Hmm, just renaming the repo itself is not enough, let's see what else I can remove.

**Comment by pieterdevriesch** [comment]
I renamed the repo and removed all documentation and descriptions, still not working. Never mind it's a very small resource and I have a few VPS that can run it as a sidejob. But maybe update the documentation :)

**Comment by ToniG22** [comment]
Oracle has free VMs I'm trying to set up my anisette server on them, altough I'm not having luck connecting. The networking is a bit messy.

**Comment by shalom-aviv** [comment]
**invalid repository** message from Render

<img width="1380" alt="Screenshot 2023-10-13 at 2 39 16 AM" src="https://github.com/Dadoum/Provision/assets/78256107/d6f3e886-f7d3-49af-8ee2-abaedc8538e6">

**Comment by ToniG22** [comment]
> **invalid repository** message from Render
> 
> <img alt="Screenshot 2023-10-13 at 2 39 16 AM" width="1380" src="https://user-images.githubusercontent.com/78256107/274755818-d6f3e886-f7d3-49af-8ee2-abaedc8538e6.png">

Render is not allowing the creation of this type of service, mostly because many people are abusing it tbh.  If you read all the coments above you would know that by now, instead of Render use Oracle Cloud, they provide a free computing unit, it's enough for anisette server hosting.
